### PR TITLE
SASS-681 - Upgraded bootstrap to version 5.3.0, added logic to propag…

### PR DESCRIPTION
**Pull request by Dula-J**
…ate headers externally

### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
 - [x]  Have you pulled the branch down?
 - [x]  Have you assigned yourself to the PR? 
 - [x]  Have you moved the task to “in review” on JIRA? 
 - [x]  Have you checked to ensure all dependencies are up to date? 

##### Whilst Reviewing
 - [x]  Have you run the tests?
 - [x]  Have you run the journey tests?
 - [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
 - [x]  Have you checked for merge conflicts?
 - [x]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR 
 - [x]  Have you run the tests?
 - [x]  Have you run the journey tests? 
 - [x]  Have you addressed warnings where appropriate? 

##### After PRs been Reviewed
 - [x]  Have you checked the PR Builder passes?
 - [x]  Have you checked for merge conflicts?
 - [x]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -5,18 +5,18 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "3.4.0",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "5.3.0",
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:8]: can be "bootstrap-frontend-play-28"

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -5,18 +5,18 @@ import sbt._
...
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-27"         % "0.50.0"
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:9]: can be "hmrc-mongo-play-28"

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.5")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
```
Comment (BenCosfordHMRC) [inline project/plugins.sbt:10]: can update play to 2.8.8 and then update any play27 dependencies to play 28

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -5,18 +5,18 @@ import sbt._
...
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "3.4.0"                % Test,
-    "org.scalatest"           %% "scalatest"                % "3.0.9"                 % Test,
+    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "5.3.0"                % Test,
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:13]: can be "bootstrap-test-play-28"

File: project/AppDependencies.scala [hunk 5]
```diff
@@ -5,18 +5,18 @@ import sbt._
...
+    "org.scalatest"           %% "scalatest"                % "3.2.9"                 % Test,
     "com.typesafe.play"       %% "play-test"                % current                 % Test,
     "org.pegdown"             %  "pegdown"                  % "1.6.0"                 % "test, it",
-    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.35.10"               % "test, it",
+    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.36.8"               % "test, it",
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:17]: In the FE found removing flexmark didnt break anything, do you know if it's needed?

File: project/AppDependencies.scala [hunk 6]
```diff
@@ -5,18 +5,18 @@ import sbt._
...
     "org.scalatestplus.play"  %% "scalatestplus-play"       % "4.0.3"                 % "test, it",
-    "com.github.tomakehurst"  %  "wiremock-jre8"            % "2.27.2"                % "test, it",
+    "com.github.tomakehurst"  %  "wiremock-jre8"            % "2.28.0"                % "test, it",
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:19]: "scalatestplus-play" version can be bumped to "5.1.0"

File: project/AppDependencies.scala [hunk 7]
```diff
@@ -5,18 +5,18 @@ import sbt._
...
     "org.scalamock"           %% "scalamock"                % "4.4.0"                 % Test,
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-27"  % "0.50.0"      % "test, it"
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:21]: scalamock can be bumped to 5.1.0 & mongo one can be hmrc-mongo-test-play-28

File: project/plugins.sbt [hunk 8]
```diff
@@ -1,16 +1,13 @@
...
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```
Comment (BenCosfordHMRC) [inline project/plugins.sbt:12]: coverage can be bumped up to 1.8.2

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 5 above)
Comment (Dula-J) [reply project/AppDependencies.scala:17]: seems to break the backends, so will keep it

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**